tests/unit/lms/services/jwt_oauth2_token_test.py

```diff
@@ -0,0 +1,86 @@
+from datetime import datetime, timedelta
+from unittest.mock import sentinel
+
+import pytest
+from freezegun import freeze_time
+
+from lms.services.jwt_oauth2_token import JWTOAuth2TokenService, factory
+from tests import factories
+
+
+class TestJWTOAuth2TokenServiceTest:
+    @freeze_time("2022-04-04")
+    def test_save_new_token(self, svc, lti_registration):
+        token = svc.save(lti_registration, "SCOPES", "ACCESS_TOKEN", 3600)
+
+        assert token.access_token == "ACCESS_TOKEN"
+        assert token.received_at == datetime(2022, 4, 4, 0)
+        assert token.expires_at == datetime(2022, 4, 4, 1)
```

Comment:
The reason for the 1 here is a little subtle. How about:

```suggestion
        assert token.expires_at == token.received_at + timedelta(seconds=3600)
```

Or something similar?